Pull

vladislav-serdyuk opened the pull request:
- теперь есть нормальная веб страница
- ещё было какое-то страное измерение фпс
- переделка под PEP8(где были силы на это)
- и ещё всякое нужное (и не нужное)

vladislav-serdyuk committed:
- теперь есть нормальная веб страница
- ещё было какое-то страное измерение фпс
- переделка под PEP8(где были силы на это)
- и ещё всякое нужное (и не нужное)

vladislav-serdyuk committed:
- теперь есть нормальная веб страница
- ещё было какое-то страное измерение фпс
- переделка под PEP8(где были силы на это)
- и ещё всякое нужное (и не нужное)

ZernovTechno commented:
pls апдейт код на новую версию, которую я только что подгрузил
ПР прикольный, но устарел софт
В новой версии сильно не изменилось

vladislav-serdyuk committed:
- теперь есть нормальная веб страница
- ещё было какое-то страное измерение фпс
- переделка под PEP8(где были силы на это)
- и ещё всякое нужное (и не нужное)

vladislav-serdyuk committed:
- теперь есть нормальная веб страница
- ещё было какое-то страное измерение фпс
- переделка под PEP8(где были силы на это)
- и ещё всякое нужное (и не нужное)
